proposals/NNNN-Add-screen-stack-management-module.md

```diff
@@ -0,0 +1,96 @@
+# Add screen stack management module
+
+* Proposal: [SDL-NNNN](NNNN-Add-screen-stack-management-module.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / iOS / Java Suite / JavaScript Suite / RPC ]
+
+## Introduction
+The purpose of this proposal is to reduce the burden on screen development for app developers by adding a screen management module.
+
+
+## Motivation
+Currently, the screens displayed by `Show` and `PerformInteraction` are independent, and there is no relation between each screen. However, in many cases of the App development process, each screen has a relationship with each other. Therefore, the screen transition specifications expected by the app developers may not be realized. By adding a screen management module and managing transitions, this problem can be solved and reduce the burden to screen development. For example, pressing the back button to enable screen transitions such as return to the previous screen. As a result, it is possible to build a better UX by reducing the inconvenience of use as much as possible for the users.
+
+<b>Current behavior</b>
+ - `PerformInteraction` and `Show (template)` are independent screens.
+ - HMI controls menu/submenu displays and screen transitions.
+ - Contents displayed in menu/submenu are provided by `AddCommand`, `DeleteCommand`, `AddSubMenu`, `DeleteSubMenu` of the app.
+ - When pushing the menu item, the HMI sends an `onCommand` notification to the mobile, and then the mobile determines the operation.
```

Comment:
Can you please specify what "mobile" refers to here?  This should be "mobile device" if it is phone-specific, "application" if it refers to app functionality, or "app library" if it refers to SDL library functionality (iOS, Java Suite, or JavaScript Suite).

Reply:
Hi, @theresalech -san.
Thank you in advance for your prompt reply.
In line 19, I will modify the word "mobile" to "SDL application" and make a commit.
Could you please confirm it again.
Thank you.